v2/api/insights/v1api20171001/structure.txt

```diff
@@ -0,0 +1,31 @@
+// Code generated by azure-service-operator-codegen. DO NOT EDIT.
+github.com/Azure/azure-service-operator/v2/api/insights/v1api20171001
+---------------------------------------------------------------------
+APIVersion: Enum (1 value)
+└── "2017-10-01"
+
+PricingPlan: Resource
```

Comment:
This is the newest PricingPlan API?

Reply:
Newest, and only, as far as I can tell.

<img width="668" height="204" alt="image" src="https://github.com/user-attachments/assets/117d8d23-db9b-4e27-b388-00a8f4aba43f" />